### PR TITLE
SR-1.5 updates

### PR DESCRIPTION
**Pull request by krogla**

### CODE REVIEW

**Comment by mkurayan** [comment]
Documentation Update Needed for `deposit-security-module.md`:

The `maxDepositsPerBlock` and `minDepositBlockDistance` parameters have been relocated from the Deposit Security Module (DSM) to the Staking Router level. The documentation should be updated to reflect these changes.

- Remove or update the descriptions of maxDepositsPerBlock and minDepositBlockDistance from the DSM documentation.
- Eliminate references to any getters and setters, such as getMinDepositBlockDistance, which were removed as part of the Staking Router 1.5 (SR-1.5) scope updates.

**Comment by mkurayan** [comment]
@vpetrenko The Sanity Checker documentation currently includes the description for oneOffCLBalanceDecreaseBPLimit and its related methods.
- Please update or remove the description for these items to align with the latest changes related to the negative rebase feature.

**Comment by mkurayan** [comment]
Add to `node-operators-registry.md` description of 
- new `decreaseVettedSigningKeysCount` method
- `updateTargetValidatorsLimits`

File: docs/guides/oracle-spec/validator-exit-bus.md [hunk 1]
```diff
@@ -45,13 +48,20 @@ A few hours later it might look like the following:
...
 
-The final exit order predicate sequence:
+The final exit order predicate sequence to fulfill withdrawal requests:
 
 1. Validator whose operator with the lowest number of delayed validators
-2. Validator whose operator with the highest number of targeted validators to exit
-3. Validator whose operator with the highest stake weight
-4. Validator whose operator with the highest number of validators
-5. Validator with the lowest index
+2. Validator whose operator with the highest number of forced targeted validators to exit
```
Comment (TheDZhon) [inline docs/guides/oracle-spec/validator-exit-bus.md:54]:
```suggestion
2. Validator whose operator with the highest number of boosted targeted validators to exit
```

File: docs/guides/oracle-spec/validator-exit-bus.md [hunk 2]
```diff
@@ -83,9 +93,9 @@ Statistics for each node operator, which are needed for sorting their validators
 
 - validators count that are not yet in CL
 - validators that are in CL and are not yet requested to exit and not on exit
-- validators that are in CL and requested to exit but not on exit and not requested to exit recently
-- target validators count
-- checks whether the target limit flag is enabled
+- validators that are in CL and requested to exit but not on exit and not requested to exit recently 
+- target type (forced/soft) and target validators count
```
Comment (TheDZhon) [inline docs/guides/oracle-spec/validator-exit-bus.md:97]:
```suggestion
- target type (soft/boosted) and target validators count
```

**Review by TheDZhon — CHANGES_REQUESTED** [review]
Looks good!

The only concern here is the forced / boosted ambiguity.

File: docs/guides/oracle-spec/validator-exit-bus.md [hunk 3]
```diff
@@ -6,13 +6,16 @@ It's advised to read [What is Lido Oracle mechanism](/guides/oracle-operator-man
...
+A report calculation consists of 6 key steps:
 
 1. Calculate withdrawals amount to cover with ether.
 2. Calculate ether rewards prediction per epoch.
-3. Calculate withdrawal epoch for next validator eligible for exit to cover withdrawal requests if needed
-4. Prepare validators exit order queue
-5. Go through the queue until the exited validators’ balances cover all withdrawal requests (considering the predicated final exited balance of each validator).
+3. Calculate withdrawal epoch for next validator eligible for exit to cover withdrawal requests if needed.
+4. Prepare validators exit order queue to fulfill withdrawals.
+5. Extend with forced to exit validators up to report limit or until there is no force requests.
```
Comment (F4ever) [inline docs/guides/oracle-spec/validator-exit-bus.md:17]:
```suggestion
5. Extend exit list with forced to exit validators (`targetLimitMode` is set to the boosted mode) up to the report limit or until there are no forced requests.
```

**Review by TheDZhon — APPROVED** [review]
GG

**Review by rkolpakov — APPROVED** [review]
LGTM

**Pull request merged**